config/acme/config_archive.xml

```diff
@@ -1,197 +0,0 @@
-<components version="2.0">
```

Comment:
Are you sure it's safe to remove these files? It looks like you may be assuming that this information has been split out into components, but I don't see config_component.xml files for any active CESM components.

Even if it is safe to remove these files: It seems the removal of this file and the similar cesm file should be in a separate PR. Or, if there is really a connection to this PR, then this removal should at least be noted in the top-level comments of the PR.

Also flagging @bertinia on this.

Reply:
According to the file config_files.xml and the entry ARCHIVE_SPEC_FILE there all config_archive.xml files are distributed to components.

Reply:
Well it looks like that's wrong:

```
[roo2:~/cesm_code/cesm2_0_alpha07]$ find . -name config_archive.xml
./cime/config/acme/config_archive.xml
./cime/config/cesm/config_archive.xml
./cime/src/components/data_comps/datm/cime_config/config_archive.xml
./cime/src/components/data_comps/dice/cime_config/config_archive.xml
./cime/src/components/data_comps/dlnd/cime_config/config_archive.xml
./cime/src/components/data_comps/docn/cime_config/config_archive.xml
./cime/src/components/data_comps/drof/cime_config/config_archive.xml
./cime/src/components/data_comps/dwav/cime_config/config_archive.xml
./cime/src/drivers/mct/cime_config/config_archive.xml
```

Reply:
Or maybe ARCHIVE_SPEC_FILE is trying to be future-proof for whenever the components do distribute those files.